### ISSUE TITLE
[spec failure] adapter_memcached(_native)_with_default_expires persists values

### ISSUE THREAD

**Issue by asppsa** [minad/moneta#130]
```
 1) adapter_memcached_native_with_default_expires persists values
     Failure/Error: store[key1].should == val1
       failure: 12 tests, on:
       ["q257OtBFZL", "oWQDSy"]
       expected: "oWQDSy"
            got: nil (using ==)
     Shared Example Group: :persist called from ./spec/helper.rb:243
     # ./spec/features/persist.rb:9:in `block (4 levels) in <top (required)>'
     # ./spec/features/persist.rb:8:in `tap'
     # ./spec/features/persist.rb:8:in `block (3 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/property.rb:33:in `call'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/property.rb:33:in `block in check'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/generator.rb:83:in `call'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/generator.rb:83:in `generate'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/property.rb:31:in `check'
     # ./spec/features/persist.rb:3:in `block (2 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     #   expected: "oWQDSy"
     #        got: nil (using ==)
     #   ./spec/features/persist.rb:9:in `block (4 levels) in <top (required)>'
```

**Comment by asppsa** [comment]
Seems to have gone away with the update to minimum MRI requirements

**Comment by asppsa** [comment]
Nope, hasn't gone away:

~~~
  1) adapter_memcached_with_default_expires persists values
     Failure/Error: store[key1].should == val1
     
       failure: 46 tests, on:
       ["sDFaJlQCw/IRBgaujnp", "VfSTuX"]
     
       expected: "VfSTuX"
            got: nil (using ==)
     Shared Example Group: :persist called from ./spec/helper.rb:274
     # ./spec/features/persist.rb:9:in `block (4 levels) in <top (required)>'
     # ./spec/features/persist.rb:8:in `tap'
     # ./spec/features/persist.rb:8:in `block (3 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.3.0/gems/rantly-1.1.0/lib/rantly/property.rb:33:in `block in check'
     # ./vendor/bundle/ruby/2.3.0/gems/rantly-1.1.0/lib/rantly/generator.rb:83:in `generate'
     # ./vendor/bundle/ruby/2.3.0/gems/rantly-1.1.0/lib/rantly/property.rb:31:in `check'
     # ./spec/features/persist.rb:3:in `block (2 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     #   expected: "VfSTuX"
     #        got: nil (using ==)
     #   ./spec/features/persist.rb:9:in `block (4 levels) in <top (required)>'
~~~

**Comment by asppsa** [comment]
This issue was caused by the fact that memcached expires entries at the beginning of each second.  See https://github.com/memcached/memcached/issues/307.

This is resolved in 2994f49f66783599475692624d521088ee49043f (and some subsequent commits) by introducing the "time resolution" and "min TTL" concepts.  For most stores, these are the same, but for stores like memcached, the min TTL needs to be double the time resolution.
